libvirt.go

```diff
@@ -94,6 +95,75 @@ type qemuError struct {
 // DomainXMLFlags specifies options for dumping a domain's XML.
 type DomainXMLFlags uint32
 
+// Consts used for flags
+
+// virDomainModificationImpact and virTypedParameterFlags
```

Comment:
can this comment be more descriptive than a reference to libvirt's source?

Reply:
Added more details, and comments for each of the consts too.